hack/arktos-up-scale-out-poc.sh

```diff
@@ -495,8 +512,8 @@ if [[ "${START_MODE}" != "kubeletonly" ]]; then
   fi
   if [ "${IS_RESOURCE_PARTITION}" != "true" ]; then
      kube::common::start_kubescheduler
-  fi
-  start_kubedns
+     start_kubedns
```

Comment:
do we not need this?

Reply:
In the tenant partition, I tried to keep things as before. So I keep it.